hw1/bashcommands.py

```diff
@@ -0,0 +1,141 @@
+from collections import deque
+from typing import Callable, TypeVar, Optional, Tuple, Union
```

Comment:
You have unused imports here.

Reply:
Oh my god, why does this thing not remind be about them  (╯°□°）╯︵ ┻━┻